Cython/Compiler/Options.py

```diff
@@ -416,6 +418,12 @@ def parse_directive_value(name, value, relaxed_bool=False):
                 name, orig_value))
     elif type is str:
         return str(value)
+    elif name=="fastcall_args":
+        # FIXME - can only take single value argument in a comment
+        if value.lower == "none":
+            return [None, None]
+        else:
+            return [parse_directive_value("", value, relaxed_bool, type=bool)]*2
```

Comment:
Why isn't `type` some kind of conversion helper or class in this case? Why does this need a special case?